Test/dafny2/SnapshotableTrees.dfy.expect

```diff
@@ -1,25 +1,7 @@
 SnapshotableTrees.dfy(71,25): Error: A precondition for this call might not hold.
```

Comment:
Is this really the same test?

Reply:
My understanding is that, previously, Z3 wasn't able to prove certain things, but that it was still a useful test. With the changes to the structure, Z3 can prove more. I could be wrong about that, but it seems plausible.